### PR TITLE
feat: Translate the website to spanish

### PR DESCRIPTION
**Pull request by diegonqui**
Feature completed

### CODE REVIEW

File: src/main/webapp/WEB-INF/jsp/vets/createOrUpdateVetForm.jsp [hunk 1]
```diff
@@ -8,21 +9,21 @@
...
         <div class="form-group">
             <div class="col-sm-offset-2 col-sm-10">
                 <c:choose>
                     <c:when test="${vets['new']}">
-                        <button class="btn btn-default" type="submit">Add Veterinary</button>
+                        <button class="btn btn-default" type="submit">Añadir veterinario</button>
                     </c:when>
                     <c:otherwise>
-                        <button class="btn btn-default" type="submit">Update Veterinary</button>
+                        <button class="btn btn-default" type="submit">Añadir/Actualizar veterinario</button>
```
Comment (marsamber) [inline src/main/webapp/WEB-INF/jsp/vets/createOrUpdateVetForm.jsp:26]:
```suggestion
                        <button class="btn btn-default" type="submit">Actualizar veterinario</button>
```

**Review by marsamber — CHANGES_REQUESTED** [review]
Actualizar ese cambio

**Comment by diegonqui** [comment]
Change accepted

**Review by marsamber — APPROVED** [review]
Genial!

**Pull request merged**